manifests/grafana-dashboardDefinitions.yaml

```diff
@@ -21957,7 +22035,7 @@ items:
                       "options": [
 
                       ],
-                      "query": "label_values(namespace_workload_pod:kube_pod_owner:relabel{namespace=~\"$namespace\", workload=~\".+\"}, workload_type)",
+                      "query": "label_values(namespace_workload_pod:kube_pod_owner:relabel{cluster=\"$cluster\",cluster=\"$cluster\",namespace=~\"$namespace\", workload=~\".+\"}, workload_type)",
```

Comment:
This is weird. I wonder why `cluster=` is specified twice? :thinking:

Reply:
I saw that, bit spooky, but should be harmless.. right?

Reply:
yeah, it is fine, but I still wonder where was this added. 

Found a source of the problem: https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/521